spec/api-shell-spec.ts

```diff
@@ -17,6 +18,15 @@ describe('shell module', () => {
     let envVars: Record<string, string | undefined> = {};
     let server: http.Server;
 
+    after(() => {
+      if (process.env.CI && process.platform === 'win32') {
+        // Edge may cause issues with visibility tests, so make sure it is closed after testing.
+        const killEdge = 'Get-Process | Where Name -Like "msedge" | Stop-Process';
+        // Asynchronously call powershell to cleanup Edge so that it doesn't cause a test timeout.
+        exec(killEdge, { shell: 'powershell.exe' });
```

Comment:
I had to use `exec` here to asychronously call powershell because `execSync` caused a timeout as seen here: https://github.com/electron/electron/actions/runs/12422052193/job/34685213121#step:18:1308